website/docs/Software/OpenBCISoftware/02_GUI_Widget_Guide.md

```diff
@@ -31,6 +31,16 @@ To change the gain of a channel, you can click a dropdown and then scroll to vie
 
 Starting with GUI v5.0.2, all ADS1299 boards (Cyton, Cyton+Daisy) will use a dynamic scaler. This ensures that the scale factor, set in the Hardware Settings view, is always in sync with the numbers shown on user interface displays and recordings. Also, users will need to stop the data stream in order to access the Hardware Settings UI inside the Time Series Widget.
 
+### Filters
+
+![image](../../assets/SoftwareImages/OpenBCISoftware/GUI_5.1.0-FiltersUI.png)
+
+The GUI contains several built in filters that can be applied to the data in real-time. These can be accessed by clicking on the `Filters` button. This will open a menu that allows you to toggle the filters on/off, choose the frequency range, type, and order of the filters. On the top right you will find the notch filter, which can be used to [reduce AC noise](Troubleshooting/01-MinimizingNoise.md#get-rid-of-ac-noise) by filtering out 50Hz or 60Hz noise. To the left of the notch filter, there is a dropdown menu to choose between applying a bandpass or bandstop filter to your data. The bandpass filter allows a specific range of frequencies to pass through while blocking or attenuating frequencies outside of that range. On the other hand, the bandstop blocks or attenuates a specific range of frequencies while allowing frequencies outside of that range to pass through. The parameters for this filter can be changed in the settings below.
```

Comment:
several built-in filters (add hyphen)

Reply:
Remove "blocks" and just use "attenuate" or "reduces". You could even link to https://en.wikipedia.org/wiki/Attenuation to help people understand what this term means.

Reply:
Drop "The parameters for this filter can be changed in the settings below." Grammarly is flagging this sentence as not great.